### PR TITLE
DCOS-16400: Replace SDK.get with direct imports

### PR DESCRIPTION
**Pull request by nLight**
**Deps**:
⚠️  Should go together with: https://github.com/mesosphere/dcos-ui-plugins-private/pull/614

This PR replaces `SDK.get()` with direct imports utilizing the `#SRC` webpack alias so that we aren't bound to a specific location.

This is an interim step for modularization. Direct imports will be replaced by a proper SKD/Loader mechanism.

Closes DCOS-16400

### CODE REVIEW

File: src/js/plugin-bridge/Loader.js [hunk 1]
```diff
@@ -1,42 +1,17 @@
-// Provide webpack contexts for smarter build. Without these,
-// webpack will try to be smart and auto create the contexts,
-// doubling the built output
```
Comment (nLight) [inline src/js/plugin-bridge/Loader.js]: I haven't checked if the build size changes with my solution.

Comment (MatApple) [inline src/js/plugin-bridge/Loader.js]: Have you had a chance to check this @nLight?

Comment (nLight) [inline src/js/plugin-bridge/Loader.js]: Not yet, will do today

**Comment by nLight** [comment]
😩  @MatApple 

master:
screen shot 2017-07-12 at 18 00 00

current:
screen shot 2017-07-12 at 18 08 23

**Comment by MatApple** [comment]
@nLight should we close this?

**Comment by nLight** [comment]
I'll do a force push soon

**Pull request closed without merging**